### ISSUE TITLE
Rescan of DB does not remove "various artist" category from corrected entries

### ISSUE THREAD

**Issue by K3str3l** [music-assistant/hass-music-assistant#313]
When incomplete ID3 tags are inplace (albumartist is blank) and initial scan is done, these artists are bucketed under "Various Artists" - this is working as intended.
When the ID3 tags are then corrected, and the DB is rescanned, the albums for those albums remain categorised under various artists.

**Comment by marcelveldt** [comment]
should be fixed in beta 6 unfortunately the existing "mismatches" will not be picked up so I'm afraid a full scan is needed to correct it

**Comment by erkr** [comment]
@K3str3l 
Maybe you have time to verify this fix by making some irrelevant changes to some files that where corrected. That should trigger the reevaluation  I guess, to see if they are removed from various artists.
Best Eric

**Comment by K3str3l** [comment]
Change some tags, do a rescan, and see if library updates to reflect those changes?

**Comment by erkr** [comment]
> Change some tags, do a rescan, and see if library updates to reflect those changes?

Yep, and check if the file is removed from various artists and reload the integration. It would be nice to see ithat works without a full rescan of the DB)

**Comment by pejotigrek** [comment]
> Change some tags, do a rescan, and see if library updates to reflect those changes?

forgive me if I'm asking for some trivial thing, but what hides behind "do a rescan" or "full rescan"?
I had a similar problem as OP, I've made some changes to the library, chosed "run sync" option from the menu and indeed, changes are being seen, but the old, unchanged albums still remains. I reloaded the integration, force-refreshed my browsers etc. and it's still there. do I need to remove/uninstall integration and reinstall it to make a clean, full rescan?
